src/nvim/eval/funcs.c

```diff
@@ -940,6 +940,55 @@ static void f_char2nr(typval_T *argvars, typval_T *rettv, FunPtr fptr)
       (const char_u *)tv_get_string(&argvars[0]));
 }
 
+// "charidx()" function
+static void f_charidx(typval_T *argvars, typval_T *rettv, FunPtr fptr)
+{
+  const char *str;
+  varnumber_T idx;
+  int countcc = false;
```

Comment:
Use the literal 0, not false, for falsy integers. `countcc` is used as an integer within this function.